qa/abstracts/src/main/java/io/camunda/zeebe/process/test/qa/abstracts/util/Utilities.java

```diff
@@ -244,7 +244,19 @@ public static final class ProcessPackTimerStartEvent {
     public static final String RESOURCE_NAME = "timer-start-event-daily.bpmn";
     public static final String TIMER_ID = "timer";
   }
-
+  public static final class ProcessPackMultipleCallActivity {
+
+    public static final String RESOURCE_NAME = "multiple-call-activity.bpmn";
+    public static final String PROCESS_ID = "multiple-call-activity";
+    public static final String CALL_ACTIVITY_ID_ONE = "callactivityOne";
+    public static final String CALLED_RESOURCE_NAME_ONE = ProcessPackAlternateStartEndEvent.RESOURCE_NAME;
+    public static final String CALLED_PROCESS_ID_ONE = ProcessPackAlternateStartEndEvent.PROCESS_ID;
```

Comment:
❌ Instead of introducing a `ProcessPackAlternateStartEndEvent` let's just use the `ProcessPackStartEndEvent`. This means you can reference the resource from the `ProcessPackStartEndEvent`. For the process id you'll need some unique string, e.g. `calledProcessIdOne`.

Reply:
I need a different processId between One and Three, to show a hasCalled(OneId) but hasNotCalled(ThreeId) on the parent project. 

Some of the requirements for my process that I have in mind: 
- Two sequential call activities, to show it correctly extracts the last called process.
- A call activity that has a call activity, to show the parent process does not detect nested child processes. (Which also means that the nested child process needs to have a different process Id than the 2 non-nested child processes.)

The process id is baked into the resource, so I made a new simple resource to prevent having to do some extra step.
Is there a way to deploy the same process twice with a different process id? 
Or is there another process that we can use for this?

Reply:
Oh sorry, my bad. I got confused with how we create processes in the Zeebe repo. You are totally right that you need the second process and resource. You can disregard these comments about removing the resource.